src/deluge/modulation/midi/midi_param_collection.cpp

```diff
@@ -279,19 +279,6 @@ bool MIDIParamCollection::mayParamInterpolate(int32_t paramId) {
 }
 
 int32_t MIDIParamCollection::knobPosToParamValue(int32_t knobPos, ModelStackWithAutoParam* modelStack) {
-
-	if (getCurrentUI()
-	    != &automationInstrumentClipView) { //let the automation instrument clip view handle the drawing of midi cc value
-		char buffer[5];
-		int32_t valueForDisplay = knobPos;
-		valueForDisplay += 64;
-		if (valueForDisplay == 128) {
-			valueForDisplay = 127;
```

Comment:
As per my other comment, what about copying this logic for the new generic popup, so 128 is painted like 127?

Reply:
Done. Capped the mod encoder's at 127 for Midi CC.